instance/models/mixins/openedx_database.py

```diff
@@ -490,6 +541,11 @@ def get_database_settings(self):
             new_settings.update(self._get_mongo_settings())
 
         # RabbitMQ:
-        new_settings.update(self._get_rabbitmq_settings())
+        # TODO: Use a real condition here
+        if False:
```

Comment:
@gabor-boros yeah we could define a boolean field on the `RedisInstanceMixin` like `uses_redis`, then:

```
if self.uses_redis:
  new_settings.update(self._get_redis_settings())
else:
  new_settings.update(self._get_rabbitmq_settings())
```